modules/ROOT/pages/shared-resources.adoc

```diff
@@ -53,11 +53,12 @@ You can find the following files when you package a domain into a JAR file:
 
 To use shared resources in your apps, you must complete the following tasks.
 
-. <<Creating a New Domain, Create a new domain>>.
-. Define one or more shared resources in that domain.
+. <<create_domain_project, Create a new domain>>.
+. <<define-shared-resources, Define one or more shared resources in that domain>>.
 . <<assoc_apps_domain, Associate applications with the domain>>.
-. <<Referencing Shared Resources, Reference shared resources>> in your applications.
-. <<Deploying with Shared Resources, Deploy>> the domain and the applications with shared resources. Note that you can also deploy domain and apps together in a <<Deploying Domain Bundles, domain bundle>>.
+. <<reference-shared-resources, Reference shared resources>> in your applications.
+. <<deploy-shared-resources, Deploy the domain and the applications with shared resources>>. +
+Note that you can also deploy domain and apps together in a <<Deploying Domain Bundles, domain bundle>>.
 
 The sections below describe the procedure for each of these steps.
```

Comment:
OMIT. We want topics to be independent, without referring to "sections below" or similar.